### PR TITLE
Added author Profile for Vijay Vavdiya

### PR DESCRIPTION
**Pull request by vvavdiya**
### Description
[Describe what this change achieves]
 Added author profile for Vijay Vavdiya
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/vvavdiya.markdown [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+- **name:** Vijay Vavdiya
+- **short_name:** Vavdiya
+- **photo:** "/assets/media/authors/vvavdiya.jpeg"
+- **github:** "vvavdiya"
+- **linkedin:** "vijayvavdiya"
```
Comment (joshuarrrr) [inline _authors/vvavdiya.markdown:6]: Remove the `**` and `""` from this section - this is front matter that will be parsed as YAML.

**Review by joshuarrrr — CHANGES_REQUESTED** [review]
Let's remove the photo for now.

**Comment by dtaivpp** [comment]
@vvavdiya please remove the picture as well! Thank you!

**Comment by vvavdiya** [comment]
@dtaivpp 
Removed profile Picture.

Thanks

**Pull request closed without merging**